src/azure-cli/HISTORY.rst

```diff
@@ -24,6 +24,31 @@ Release History
 
 * Fix `vm create` failure in Azure Stack profile.
 
+**ServiceFabric**
```

Comment:
You have to update release note in history.rst at right position.

Reply:
fixed